docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -34,8 +34,7 @@ openstack volume backup list --all-projects -c ID -c Status |grep -E '\| .+ing \
 openstack share list --all-projects -c ID -c Status |grep -E '\| .+ing \|'| grep -vi error
 openstack image list -c ID -c Status |grep -E '\| .+ing \|'
 ----
-* Collect the services topology-specific configuration before stopping services required to gather it live. You need it to compare it with the post-adoption values later on. For more information, see xref:proc_retrieving-services-topology-specific-configuration_adopt-control-plane[Retrieving services from a topology specific-configuration].
-//kgilliga: Question for Mathieu and Jiri: Should the "Retrieving services from a topology specific-configuration" module be moved to the beginning of the "Migrating databases to the control plane chapter?".
+* Collect the services topology-specific configuration before stopping services required to gather it live. You need it to compare it with the post-adoption values later on. For more information, see xref:proc_retrieving-topology-specific-service-configuration_adopt-control-plane[Retrieving topology-specific service configuration].
```

Comment:
```suggestion
* Collect the services topology-specific configuration before stopping the services that are required to gather it live. You need to compare the topology-specific service configuration with the post-adoption values after adoption is complete. For more information, see xref:proc_retrieving-topology-specific-service-configuration_adopt-control-plane[Retrieving topology-specific service configuration].
```
Or should this say, "Collect the topology-specific service configuration before stopping..."?
I'm also unsure about whether my edit, "after adoption is complete", is accurate. Feel free to reject.

Reply:
The topology-specific config gathering actually is later than here in the Adoption guide, so i think this prerequisite step does not apply. We need MariaDB running for the "Retrieving topology-specific service configuration" step but i don't think we need Nova running. @bogdando or @gibizer can you confirm?

Reply:
If confirmed, i would remove this prerequisite bullet point.

Reply:
Discussed on Slack, given that we use `podman exec` on `nova_api` container, we do need to retrieve the config before stopping the services. So i went with an ordering fix instead:

https://github.com/openstack-k8s-operators/data-plane-adoption/pull/453/commits/98bef2bd709de452087e0624d2da31d3644bb710